crates/biome_js_formatter/src/comments.rs

```diff
@@ -1244,6 +1245,32 @@ fn handle_import_export_specifier_comment(
     }
 }
 
+/// Ensure that comments before the `async`` keyword are placed just before it.
```

Comment:
Could you please check if we need to do the same thing for generators?
```
class Foo {
	@dec /*before*/
	/*after*/ * method() {}
}
```

Reply:
Generators have the same issue. I will submit another PR to fix it.